### ISSUE TITLE
Missing AccessibilityLabel for Info button in a chat view

### ISSUE THREAD

**Issue by MarcoZehe** [deltachat/deltachat-ios#629]
- Device: iPhone or iPad
- iOS version: 13.4
- Delta Chat version: 1.3 app store or 1.5 TestFlight
- Expected behavior: When opening a chat window, like the greeting message, and exploring the items with VoiceOver, the Info button should be labeled with "Information" or "Contact details" or something appropriate.
- Actual behavior: When reaching the button with VoiceOver, VoiceOver says "button", sometimes followed by "May be Information".
- Steps to reproduce the problem: Install Delta Chat, start VoiceOver via Settings/Accessibility/VoiceOver, or by configuring the Accessibility Shortcut to launch and shut down VoiceOver, open a chat and swipe left and right through the controls.
- Screenshots: N/A
- Logs: Not applicable, no crash, just missing information (AccessibilityLabel on the info button).

**Comment by cyBerta** [comment]
Thanks @MarcoZehe for the detailed bug description!

**Comment by nayooti** [comment]
@MarcoZehe Thank you for reporting the issue.
I tried to reproduce it and unfortunately it is not clear to me which UI element you mean by _info button_. 
Could you please by change specify which UI element you refer to. A screenshot would be a big help.

**Comment by MarcoZehe** [comment]
The button in the top right corner of a chat window, for example the sample conversation that gets added when doing a fresh install and not yet having something in one's account that's compatible with Delta Chat. That's still the case with my regular e-mail account. To the right of the header for that view is an unlabeled button, up there in the top right corner.

**Comment by nayooti** [comment]
@MarcoZehe Ok, thank you. Now I found it!
